libs/coin-modules/coin-stellar/src/network/horizon.ts

```diff
@@ -38,12 +38,8 @@ const TRESHOLD_LOW = 0.5;
 const TRESHOLD_MEDIUM = 0.75;
 const FETCH_LIMIT = 100;
 const currency = getCryptoCurrencyById("stellar");
-let server: Horizon.Server | undefined;
 const getServer = () => {
-  if (!server) {
-    server = new Horizon.Server(coinConfig.getCoinConfig().explorer.url);
-  }
-  return server;
+  return new Horizon.Server(coinConfig.getCoinConfig().explorer.url);
```

Comment:
Maybe we can store in a local map by network ? As you said it's a bit scary performance wise

Reply:
OK, I did a map by URL in b7023c5